composer.json

```diff
@@ -19,6 +25,8 @@
         "symfony/security-bundle": "~2.4"
     },
     "suggest": {
+        "symfony/psr-http-message-bridge": "To use the PSR-7 converters",
+        "zendframework/zend-diactoros": "To use the Zend Diactoros factory",
```

Comment:
not sure if this is necessary since you have this suggest already in the bridge:
https://github.com/dunglas/psr-http-message-bridge/blob/wip/composer.json#L27